README.md

```diff
@@ -144,7 +154,7 @@ type GoCloak interface {
 	CreateUser(ctx context.Context, token, realm string, user User) (string, error)
 	CreateGroup(ctx context.Context, accessToken, realm string, group Group) (string, error)
 	CreateChildGroup(ctx context.Context, token, realm, groupID string, group Group) (string, error)
-	CreateClientRole(ctx context.Context, accessToken, realm, clientID string, role Role) (string, error)
+	CreateClientRole(ctx context.Context, accessToken, realm, clientid string, role Role) (string, error)
```

Comment:
as the interface in the readme is a copy of the interface itself located in gocloak.go, renaming some of these properties here is not correct, although it does not really do any harm, we should keep it that way